package.json

```diff
@@ -24,22 +25,31 @@
     "babel-loader": "^7.0.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "body-parser": "^1.17.2",
```

Comment:
This is something that I learned recently, but it's good practice to not use ^ to have a set version for each dependency.